docs/how-to/h-manage-app.md

```diff
@@ -57,6 +57,49 @@ unit-data-integrator-0:
     started: 2023-01-27 14:22:51 +0000 UTC
 ```
 
+## Password rotation
```

Comment:
**praise:** This is a **REALLY** well written and clear piece of docs, really like it!